diff --git a/config.h b/config.h
index a02c0ca..09b7020 100644
--- a/config.h
+++ b/config.h
@@ -11,22 +11,18 @@
 #define CLICKABLE_BLOCKS 1
 
 // Control whether a leading delimiter should be prepended to the status.
-#define LEADING_DELIMITER 0
+#define LEADING_DELIMITER 1
 
 // Control whether a trailing delimiter should be appended to the status.
-#define TRAILING_DELIMITER 0
+#define TRAILING_DELIMITER 1
 
 // Define blocks for the status feed as X(cmd, interval, signal).
 #define BLOCKS(X)         \
-    X("sb-mail", 600, 1)  \
-    X("sb-music", 0, 2)   \
-    X("sb-disk", 1800, 3) \
-    X("sb-memory", 10, 4) \
-    X("sb-loadavg", 5, 5) \
-    X("sb-mic", 0, 6)     \
-    X("sb-record", 0, 7)  \
-    X("sb-volume", 0, 8)  \
-    X("sb-battery", 5, 9) \
-    X("sb-date", 1, 10)
+    X("sb-keyboard-locks", 0, 15)   \
+    X("sb-connection", 0, 14)   \
+    X("sb-brightness", 0, 16)   \
+    X("sb-volume", 0, 13)   \
+    X("sb-battery", 180, 12) \
+    X("sb-date", 60, 11)   \
 
 #endif  // CONFIG_H
