diff --git a/tmux-power.tmux b/tmux-power.tmux
index bc09440..eb54482 100755
--- a/tmux-power.tmux
+++ b/tmux-power.tmux
@@ -82,7 +82,7 @@ G11=#6c6c6c #242
 G12=#767676 #243
 
 FG="$G10"
-BG="$G04"
+BG="colour0"
 
 # Status options
 tmux_set status-interval 1
@@ -132,7 +132,8 @@ fi
 if [[ $prefix_highlight_pos == 'R' || $prefix_highlight_pos == 'LR' ]]; then
     RS="#{prefix_highlight}$RS"
 fi
-tmux_set status-right "$RS"
+#tmux_set status-right "$RS"
+tmux_set status-right "#{prefix_highlight}"
 
 # Window status format
 tmux_set window-status-format         "#[fg=$BG,bg=$G06]$rarrow#[fg=$TC,bg=$G06] #I:#W #[fg=$G06,bg=$BG]$rarrow"
