diff --git a/config.mk b/config.mk
index 69b8092..3c28966 100644
--- a/config.mk
+++ b/config.mk
@@ -46,7 +46,7 @@ XRENDER = -lXrender
 #XEXTLIB = -lXext
 
 # Uncomment this for the swallow patch / SWALLOW_PATCH
-#XCBLIBS = -lX11-xcb -lxcb -lxcb-res
+XCBLIBS = -lX11-xcb -lxcb -lxcb-res
 
 # This is needed for the winicon and tagpreview patches / BAR_WINICON_PATCH / BAR_TAGPREVIEW_PATCH
 #IMLIB2LIBS = -lImlib2
